features/step_definitions/verification_process_steps.rb

```diff
@@ -121,6 +121,10 @@ def user(*traits)
   find(:xpath, IvlDocumentsPage.view_history_option).click
 end
 
+And(/^Individual clicks on verify$/) do
+  find(:xpath, IvlDocumentsPage.verify_option).click
```

Comment:
we should not be using xpath at all anymore

Reply:
fixed